<launch>
  <node name="servo_joint_state_publisher" pkg="roomac_arm" type="servo_joint_state_publisher_node.py" output="log" ns="arm_position_controller">
    <rosparam command="load" file="$(find roomac_arm)/config/arm_controller_config.yaml" />
  </node>

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
    <rosparam param="source_list">["/arm_position_controller/joint_states_arm_stub", "/wheel_joint_publisher/joint_states_wheels"]</rosparam>
    <param name="rate" type="double" value="10.0" />
  </node>
</launch>